README.rst

```diff
@@ -38,6 +38,7 @@ If you want to get involved head over to GitHub to get the source code, look at
 * `GitHub page`_
 * `Development documentation`_
 * `Development IRC`_
+* `Development Discord`_
```

Comment:
This could be removed here, so discord is only documented under the 'development' docs to further prevent anyone just using one of the first links they see to send support requests